src/tc/lib/src/replica.rs

```diff
@@ -501,6 +501,35 @@ pub unsafe extern "C" fn tc_replica_import_task_with_uuid(
     )
 }
 
+#[ffizz_header::item]
+#[ffizz(order = 902)]
+/// Delete a task.  The task must exist.  Note that this is different from setting status to
+/// Deleted; this is the final purge of the task.
+///
+/// Deletion may interact poorly with modifications to the same task on other replicas. For
+/// example, if a task is deleted on replica 1 and its description modified on replica 1, then
```

Comment:
```suggestion
/// example, if a task is deleted on replica 1 and its description modified on replica 2, then
```

I think this is what you meant.